website/src/pages/downloads.tsx

```diff
@@ -204,31 +204,46 @@ const Docker = ({ version, name }) => {
     return (
         <Tabs>
             <TabItem value="quay" label="quay.io">
+                <p>Main Image Distribution</p>
```

Comment:
```suggestion
                <p>Alpine Image Distribution</p>
```

Maybe its my ex-Red Hat bias, but I'm not sure I'd call the Alpine one `main`. That said, there is an argument for consistency (distro name + distro name) or libc implementation (`musl image distribution` vs `glibc image distribution`) ;-) 

Your call :-)

Reply:
The rest of the work is in #1087 if you want to take that on too :-)